Resources/doc/9e-integrating_several_parsers.md

```diff
@@ -0,0 +1,48 @@
+Step 9e: Integrating several parsers together
+=============================================
+
+Purpose
+--------
+Imagine that you need to integrate ExerciseHTMLPurifierBundle parser, bbcode parser, any other parser... it's better to have several parsers where each is responsible for it's action. That's why it's better to make chain of parsers, where order is important and parsers works one after other.
+
+Usage
+-----
+
+First, configure parser classes as services.
+
+```yaml
+    # See 9a-markup_htmlpurifier.md
+    markup.exercise_html_purifier:
+        class: FOS\CommentBundle\Markup\HtmlPurifier
+        arguments: [ @exercise_html_purifier.default ]
+        
+    # Your own parser    
+    markup.my_parser:
+        #This is your own parser that implements ParserInterface
+        class: MyBundle\Parser\MyParser
+        # If you don't plan to use this parser somewhere in the code, it's better to make it private.
+        # this should increase service container performance
+        public: false
+```
+
+Then, you have to define PipelineParser as a service and configure it with parsers.
+
+```yaml
+    markuper.pipeline_parser:
+        class: FOS\CommentBundle\Markup\PipelineParser
+        calls:
+            - [addToPipeline, ["@markup.exercise_html_purifier"]]
+            - [addToPipeline, ["@markup.my_parser"]]
+```
+
+The order of parsers is important. First parser that has been added to pipeline is executed first.
```

Comment:
``to the pipeline``.

And I think we could rephrase the whole sentence:

```
Parsers are executed in the order in which they are added to the pipeline.
```

What do you think about it ?